package-lock.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Updog",
+  "lockfileVersion": 2,
+  "requires": true,
+  "packages": {}
+}
```

Comment:
Seems like you accidentally ran npm install on the root directory, can you delete this file. Thx